### PR TITLE
Replace optimaize with OpenNLP language detector [run-systemtest]

### PR DESCRIPTION
**Pull request by jonmv**
@bratseth please review.

### CODE REVIEW

File: linguistics/src/main/java/com/yahoo/language/opennlp/LangDetectModel.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.language.opennlp;
+
+import opennlp.tools.langdetect.LanguageDetectorModel;
+
+/**
+ * @author jonmv
```
Comment (bratseth) [inline linguistics/src/main/java/com/yahoo/language/opennlp/LangDetectModel.java:7]: Could you add some javadoc here?

File: linguistics/src/main/java/com/yahoo/language/opennlp/LangDetectModel.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+ */
+public interface LangDetectModel {
+
+    LanguageDetectorModel load();
```
Comment (bratseth) [inline linguistics/src/main/java/com/yahoo/language/opennlp/LangDetectModel.java:11]: ... and here (especially, what does retuning null mean and can this throw an exception.

**Review by bratseth — APPROVED** [review]
Great

**Comment by jonmv** [comment]
Fixed the bad mail regex as well, since that was what triggered this initially.

**Comment by jonmv** [comment]
PTAL

**Review by bratseth — APPROVED** [review]
👍 👍 👍

**Comment by bratseth** [comment]
Fails with 
```
Could not create a component with id 'com.yahoo.language.opennlp.LangDetectModel183'. Tried to load class directly, since no bundle was found for spec: com.yahoo.language.opennlp.LangDetectModel183. If a bundle with the same name is installed, there is a either a version mismatch or the installed bundle's version contains a qualifier string."
15:41:53 Stacktrace was: java.lang.IllegalArgumentException: Could not create a component with id 'com.yahoo.language.opennlp.LangDetectModel183'. Tried to load class directly, since no bundle was found for spec: com.yahoo.language.opennlp.LangDetectModel183. If a bundle with the same name is installed, there is a either a version mismatch or the installed bundle's version contains a qualifier string.
15:41:53 	at com.yahoo.osgi.OsgiImpl.resolveFromClassPath(OsgiImpl.java:71)
15:41:53 	at com.yahoo.osgi.OsgiImpl.resolveClass(OsgiImpl.java:62)
15:41:53 	at com.yahoo.container.di.Container.addNodes(Container.java:200)
15:41:53 	at com.yahoo.container.di.Container.createComponentsGraph(Container.java:189)
15:41:53 	at com.yahoo.container.di.Container.getConfigAndCreateGraph(Container.java:117)
15:41:53 	at com.yahoo.container.di.Container.getNewComponentGraph(Container.java:76)
15:41:53 	at 
```

**Comment by jonmv** [comment]
Yup. Dependencies are the hardest part :<

**Comment by jonmv** [comment]
@bjorncs I gave up using a different module for this, FYI.

**Review by bjorncs — COMMENTED** [review]
👍

**Pull request merged**